contributing/code_of_conduct_concrete_example_document.rst

```diff
@@ -0,0 +1,16 @@
+Code of Conduct: Concrete Example Document
+==========================================
+This is a living document that serves to give concrete examples to unwanted behaviour. These examples have all taken place somewhere in the PHP community in the past, and are clear code of conduct violations according to the Symfony code of conduct.
+
+Concrete Examples
+-----------------
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of ‘dead’ (rejected) names
```

Comment:
The `deadname` concept is not that well-known, so what do you think about linking it to https://en.wiktionary.org/wiki/deadname or adding a brief description like this one?

```rst
- Deliberate misgendering or use of deadnames (which are the original names given to
  transgender people before initiating their transition).
```

Reply:
Changed :)